sdcm/nemesis.py

```diff
@@ -576,7 +576,7 @@ def disrupt_truncate(self):
             self.target_node.remoter.run(stress_cmd, verbose=True, ignore_status=True)
 
         # do the actual truncation
-        self.target_node.run_cqlsh('TRUNCATE {}.{}'.format(keyspace_truncate, table))
+        self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(keyspace_truncate, table), timeout=120)
```

Comment:
There is "connect_timeout" parameter. I suggest to increase it also

Reply:
connect timeout is how long does it take to connect to SQL port. Default is 5 seconds and it is alot. If CQL port doesn't answer for 5 seconds something bad happening in the system